redwood-layout-api/build.gradle

```diff
@@ -12,6 +12,7 @@ kotlin {
   sourceSets {
     commonMain {
       dependencies {
+        api projects.redwoodLayoutApiDensity
```

Comment:
What's the motivation for a different API module for these?

Reply:
Actually I suspect we'll be moving these all into Redwood's core since we want to expose things like screen information, container size, etc. as a composition local and also through `HostConfiguration` in Treehouse.

Reply:
I created a separate module so we can expose `Dp` and `Margin` to the guest code, but don't need to depend on the rest of the layout API (i.e. `MainAxisAlignment`, `CrossAxisAlignment`, etc.)